perllib/FixMyStreet/Cobrand/Bexley/Waste.pm

```diff
@@ -1081,7 +1081,8 @@ sub in_cab_logs_reason_prefixes {
         'Paper & Card' => ['PA-1100', 'PA-1280', 'PA-140', 'PA-240', 'PA-55', 'PA-660', 'PA-940', 'PC-180', 'PC-55'],
         'Food' => ['FO-140', 'FO-23'],
         'Garden' => ['GA-140', 'GA-240'],
-        'Plastics & Glass' => ['GL-1100', 'GL-1280', 'GL-55', 'GL-660', 'PG-1100', 'PG-1280', 'PG-240', 'PG-360', 'PG-55', 'PG-660', 'PG-940', 'PL-1100', 'PL-1280', 'PL-140', 'PL-55', 'PL-660', 'PL-940'],
+        'Plastics & Glass' => ['PG-1100', 'PG-1280', 'PG-240', 'PG-360', 'PG-55', 'PG-660', 'PG-940', 'PL-1100', 'PL-1280', 'PL-140', 'PL-55', 'PL-660', 'PL-940'],
+        'Glass' => ['GL-1100', 'GL-1280', 'GL-55', 'GL-660', 'GL-240'],
```

Comment:
If GL-240 is added here, I think it needs to be added to `image_for_unit()` and `_containers` too.

I think what has caused confusion, for me at least, is that the [spreadsheet](https://3.basecamp.com/4020879/buckets/35109031/todos/6820596548#__recording_6828438129) (and surrounding conversation) mention 'No longer needed' services - which include GL-240, GL-140, and GL-Node (out of the GL-* types).

Reply:
Ah, good catch! I've removed GL-240 in a fixup commit, seemed easiest for the time being. I'll let them know about the "No longer needed" services and see what they say.